backend/pom.xml

```diff
@@ -47,6 +47,12 @@
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.4</version>
```

Comment:
Lombok is part of spring-boot-dependencies and doesn't require version to be specified. Also use `compile` scope for this dependency, cause we don't need lombok in runtime, only for compilation process. See [Maven dependency scope guide](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Scope)